### PR TITLE
Improve POST_RELEASE steps

### PR DESCRIPTION
**Pull request by nikku**
* enable strict error handling in scripts (https://github.com/bpmn-io/dmn-js/commit/96e409c330b26808ba8110b13f47646d0dfa17b2)
* correct credentials used (https://github.com/bpmn-io/dmn-js/commit/bb0686e8ffcbf40430528225b215b4e8866b2406)

Moving to organization provided credentials, a thing we adopted for `@bpmn-io/form-js` already, too.

---

Closes https://github.com/bpmn-io/dmn-js/issues/645.

### CODE REVIEW

File: tasks/stages/update-demo [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+set -euo pipefail
```
Comment (barmac) [inline tasks/stages/update-demo:3]: Is it necessary? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference

File: tasks/stages/update-demo [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+shopt -s inherit_errexit nullglob
```
Comment (barmac) [inline tasks/stages/update-demo:4]: What's this for? 🤡

File: tasks/stages/update-demo (hunk 1 above)
Comment (nikku) [reply tasks/stages/update-demo:3]: It is, as I can run the same script locally. In that case it should exhibit the same behavior, too.

File: tasks/stages/update-demo (hunk 2 above)
Comment (nikku) [reply tasks/stages/update-demo:4]: Did some research and decided to settle down on that configuration [based on this article](https://dev.to/banks/stop-ignoring-errors-in-bash-3co5).

Comment (nikku) [reply tasks/stages/update-demo:4]: My goal is to catch as many errors as possible, hence that additional line (described in the article above, too).

Comment (barmac) [reply tasks/stages/update-demo:4]: Thanks for sharing knowledge.

**Review by barmac — APPROVED** [review]

**Pull request merged**